packages/apps/app-dashboard/src/config/env.ts

```diff
@@ -28,12 +28,12 @@ export const env = createEnv({
     VITE_WALLETCONNECT_PROJECT_ID: z.string(),
     VITE_LIT_PAYER_SECRET_KEY: z.string(),
     VITE_LIT_RELAY_API_KEY: z.string(),
+    VITE_VINCENT_YELLOWSTONE_RPC: z.string(),
```

Comment:
```suggestion
    VITE_VINCENT_YELLOWSTONE_RPC: z.url(),
```
Not super important, but we could start using [.url](https://zod.dev/api?id=urls)